### PR TITLE
[RJSF Add validation to the RJSF form | CU-1zgbtk0

### PR DESCRIPTION
**Pull request by humblenginr**
**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]

File: ui/components/MesheryMeshInterface/PatternService/RJSF.js [hunk 1]
```diff
@@ -159,6 +159,9 @@ function RJSF(props) {
           ObjectFieldTemplate={CustomObjFieldTemplate}
           additionalMetaSchemas={[JS4]}
           uiSchema={uiSchema(jsonSchema)}
+          liveValidate
+          showErrorList={false}
```
Comment (Abhishek-kumar09) [inline ui/components/MesheryMeshInterface/PatternService/RJSF.js:163]: okay so liveValidation is live now. ErrorList is bad for forms now? should we show it on long run? like a red badge that lists out all the error if someone clicks on it.

**Review by Abhishek-kumar09 — APPROVED** [review]
LGTM, additionally did you forget running eslint.

File: ui/components/MesheryMeshInterface/PatternService/RJSF.js (hunk 1 above)
Comment (sudo-NithishKarthik) [reply ui/components/MesheryMeshInterface/PatternService/RJSF.js:163]: yes. That's the plan

**Comment by l5io** [comment]
Task linked: [CU-1zgbtk0 Validate Designs Before Saving](https://app.clickup.com/t/1zgbtk0)

**Comment by warunicorn19** [comment]
@sudo-NithishKarthik ready to merge?

**Comment by sudo-NithishKarthik** [comment]
yes @warunicorn19

File: ui/components/MesheryMeshInterface/PatternService/RJSF.js (hunk 1 above)
Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternService/RJSF.js:163]: Goooooood

**Pull request merged**